internal/gitlab/gitlab.go

```diff
@@ -51,6 +59,72 @@ func Init(_host, _user, _token string) {
 	token = _token
 	lab = gitlab.NewClient(nil, token)
 	lab.SetBaseURL(host + "/api/v4")
+	configDirs := configdir.New("zaquestion", "lab-cli")
+	cacheDir = configDirs.QueryCacheFolder()
```

Comment:
Does this cache stuff actually belong in the GitLab package? Should it be in its own package?